config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -163,7 +163,7 @@ presubmits:
     extra_refs:
       - org: kubernetes-sigs
         repo: cluster-api-provider-azure
-        base_ref: release-1.7
+        base_ref: release-1.8
```

Comment:
Successful test run after this change:

- https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-cloud-provider-azure-e2e-capz/1634187421417476096